### DISCUSSION TITLE
[Performances test WebRTC] Deplay or packet loss is too high

### DISCUSSION THREAD

**Discussion by hugmahit** [OvenMediaLabs/OvenMediaEngine#1181] [Help]
**Describe the bug**
- Performances test with Webrtc by OvenRtc Tester
- Edge high CPU, this CPU usage is normal?
- deplay or packet loss is too high

**To Reproduce**
1. Set Edge as follows [Edge.xml](https://gist.github.com/hugmahit/7416b91aec185789f3359cac7aab8a8d)
2. Set Origin as follows [Origin.xml](https://gist.github.com/hugmahit/46a5e7f3e3d29d5193f644abf5297275)
3. With Encoder OBS
![Capture](https://user-images.githubusercontent.com/93419021/231050536-ae4cc6f5-433a-4d8b-a05f-e5e260a82ff6.PNG)
4. Test ws://domain:8333/app/stream by OvenPlayer demo

**Server (please complete the following information):**
- OS: 20.04.1 LTS (GNU/Linux 5.4.0-42-generic) - 48 cores
- OvenMediaEngine Version: [#610338b](https://github.com/AirenSoft/OvenMediaEngine/tree/610338ba196f287f0f6204c6f9bcd1d53e6fa865)
- Branch: master

**Additional context**
top -H -p [pid]:
![Screenshot (70)](https://user-images.githubusercontent.com/93419021/231051303-3036ce60-b254-4dd2-9f6a-bed3d5791dd2.png)

OvenRtcTester: Test with ws://domain:8333/app/stream ~ 500 players 
![Screenshot (68)](https://user-images.githubusercontent.com/93419021/231051276-56a8e6f1-2e6b-476c-935a-b1cb988d6267.png)

**Comment by getroot** [comment]
If it is correct that 500 sessions are connected, the cpu usage seems higher than my development environment.  The most cpu consuming task in webrtc transmission is srtp encryption.  what is your cpu?

**Reply by hugmahit** [reply]
@getroot 
CPU: Intel(R) Xeon(R) Silver 4214 CPU @ 2.20GHz

**Reply by getroot** [reply]
The CPU of my development server is AMD Ryzen 9 5900X.
I set the video bitrate to 1Mbps just like you. StreamWorker is set to 16, but CPU usage is lower than your environment. It seems to be a clock speed difference, but I'll check later if it's an issue with the hardware encryption module.

![image](https://user-images.githubusercontent.com/17779085/231352358-a7f9188a-9841-4b15-84bb-2c539c3dd5b3.png)

Although unlikely, have you ever tuned a Linux network?

https://airensoft.gitbook.io/ovenmediaengine/troubleshooting#5-2.-tuning-your-linux-kernel

**Reply by hugmahit** [reply]
@getroot 
Thank you!
I will try tunning a Linux network. If there is any problem I will report back to you soon.

**Reply by hugmahit** [reply]
@getroot 
I tried but the problem still occurs.
/etc/sysctl.conf:
![code](https://user-images.githubusercontent.com/93419021/231627166-21c484bc-9573-4547-845c-d62c5470a1a5.png)

/etc/security/limits.conf:
![code1](https://user-images.githubusercontent.com/93419021/231627379-d70d135c-d2ed-4bac-a9ab-299d29909ddf.png)
